src/components/PrecinctTallyReport.tsx

```diff
@@ -54,14 +60,14 @@ const PrecinctTallyReport = ({
   return (
     <Report>
       <h1>
-        <NoWrap>{precinct.name}</NoWrap> <NoWrap>{election.title}</NoWrap>{' '}
+        <NoWrap>{precinct.name}</NoWrap> <NoWrap>{election.title}</NoWrap>
         <NoWrap>Tally Report</NoWrap>
       </h1>
       <p>
         This report should be <strong>{reportPurpose}</strong>.
       </p>
       <p>
-        {isPollsOpen ? 'Polls Closed' : 'Polls Opened'} and report printed at:{' '}
+        {isPollsOpen ? 'Polls Closed' : 'Polls Opened'} and report printed at:
```

Comment:
Please confirm that there is still a space after the `:` in the rendered content.

Reply:
Turns out this was due to a bug that's now fixed: https://github.com/yannickcr/eslint-plugin-react/pull/2437. I'll update `eslint-plugin-react` in your PR.